colibre/scripts/gas_smoothing_lengths.py

```diff
@@ -8,7 +8,7 @@
 from swiftsimio import load
 from unyt import unyt_quantity, dimensionless, Mpc
 
-h_bounds = [1e-2, 5e2]  # comoving kpc
+h_bounds = [1e-3, 5e2]  # comoving kpc
```

Comment:
If the minimal smoothing length is 1 percent of the gravitational softening which is, say, 0.2 ckpc, then the h_min's value is going to 0.2 * 0.01 =  2e-3; we want to capture it in the plot.